packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -8,6 +8,16 @@ import 'assertions.dart';
 import 'platform.dart';
```

Comment:
Wondering if `debug.dart` is the right place for this. Maybe we should have a `release.dart` in here analogous to the already existing `profile.dart` and `debug.dart`?

Reply:
Or `constants.dart`, like we have in other modules?

It just seemed like overkill to make a new file for a single constant, and is related to debug in the sense that it is usually used in code differentiating between release and debug.

Reply:
`constants/dart` also sounds good.

It's differentiating between release and not-release (which also included profile), that's why I feel `debug.dart` is not the right place. Creating a new file shouldn't cost us anything (and users will import `flutter/foundation.dart`, so they wouldn't see that this is in a separate file.

Reply:
OK, moved to constants.dart.